source/unified-test-format/tests/invalid/clientEncryption-missing_kmsProviders.yml

```diff
@@ -0,0 +1,16 @@
+description: clientEncryptionOpts-missing_kmsProviders
```

Comment:
This test file is redundant with `clientEncryptionOpts-missing_kmsProviders`, which I suggested renaming to `clientEncryptionOpts-kmsProviders-required`.